apps/E2E/src/Menu/pages/MenuPageObject.ts

```diff
@@ -4,23 +4,20 @@ import {
   MENUITEM_NO_A11Y_LABEL_COMPONENT,
   HOMEPAGE_MENU_BUTTON,
   MENUITEM_TEST_COMPONENT,
-  MENUPOPOVER_TEST_COMPONENT,
   MENUITEM_DISABLED_COMPONENT,
   MENUITEM_FOURTH_COMPONENT,
   MENU_CALLBACK_RESET_BUTTON,
   MENUITEM_CALLBACK_LABEL,
 } from '../../../../fluent-tester/src/TestComponents/Menu/consts';
 import { BasePage, By } from '../../common/BasePage';
-import { Keys, Attribute, AttributeValue } from '../../common/consts';
-
-/* This enum gives the spec file an EASY way to interact with SPECIFIC UI elements on the page.
- * The spec file should import this enum and use it when wanting to interact with different elements on the page. */
-export const enum MenuComponentSelector {
-  MenuTrigger = 0, //this._primaryComponent
-  FirstMenuItem,
-  SecondMenuItem,
-  ThirdMenuItem,
-  FourthMenuItem,
+import { Keys } from '../../common/consts';
+
+/** Convenience enum for an external caller to get individual menu items using .getMenuItem() */
```

Comment:
Nit: Remove word "Convenience". 

"Allows caller to get specific menu items using .getMenuItem()"